Article/Article.js

```diff
@@ -1,20 +1,35 @@
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(domElement,index) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+    this.index = index;
+    // create close button
+    this.button = document.createElement('button');
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = document.querySelectorAll(`.expandButton`);
     // Using your expandButton reference, update the text on your expandButton to say "expand"
+    this.expandButton[this.index].innerText = 'Read';
     
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+   this.expandButton[this.index].addEventListener('click', (e)=>  this.expandArticle(e.target))
```

Comment:
This is great and it's working properly, however you could improve your code using the domElement you declared on line 6. 
For instance on line 11 you could just target the domElement passed in your constructor (an article in this case) instead of the document itself in this fashion `this.expandButton = this.domElement.querySelector('.expandButton');`
In this way you don't need any index to target the right button,because you are already have the right one.